### PR TITLE
Completed react-hot-button-component

### PR DESCRIPTION
**Pull request by kevin-l1**
image
image

### CODE REVIEW

File: react-hot-button-component/my-app/src/HotButton.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  const [counter, setCounter] = useState(0);
+
+  let color;
+  let textColor;
+
+  function handleClick() {
+    setCounter(counter + 1);
+  }
+
+  console.log(counter);
```
Comment (shawnkost) [inline react-hot-button-component/my-app/src/HotButton.js]: Remove all console logs before submitting!

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Kevin!

File: react-hot-button-component/my-app/src/HotButton.js (hunk 1 above)
Comment (kevin-l1) [reply react-hot-button-component/my-app/src/HotButton.js]: Oh, thought I should leave it there so you could see what my counter was. Though I guess I should have shown the actual counter inside the app

File: react-hot-button-component/my-app/src/HotButton.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  }
+
+  return (
+    <button
+      style={{ background: color, color: textColor }}
+      className="button"
+      onClick={handleClick}>
+      Hot Button
+    </button>
+  );
```
Comment (shawnkost) [inline react-hot-button-component/my-app/src/HotButton.js]: You should also be showing the amount of clicks on the screen!

**Review by shawnkost — CHANGES_REQUESTED** [review]
One more change here! Thanks for pointing that out!

File: react-hot-button-component/my-app/src/HotButton.js (hunk 1 above)
Comment (shawnkost) [reply react-hot-button-component/my-app/src/HotButton.js]: Ahh you're right, I didn't even catch that! Need to include it in your JSX

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**